ui/app/AppLayouts/Chat/views/CreateChatView.qml

```diff
@@ -29,6 +29,15 @@ Page {
         color: Theme.palette.statusAppLayout.rightPanelBackgroundColor
     }
 
+    QtObject {
+        id: d
+        property var ownContact: ContactListItemDelegate {
```

Comment:
That's wrong.. :eyes: What if the internal implementation of MembersSelector is changed?

Reply:
wrong? I don't think it's that dramatic - I guess it will be only a matter of updating. do you have any other suggestion?

Reply:
It is indeed dramatic because we shouldn't make our code tightly coupled, where one innocently looking change breaks other places. As a client, you shouldn't rely on the internals of the `MembersSelector` component. 

This is my suggestion: https://github.com/status-im/status-desktop/pull/7990

Reply:
@osmaczko thanks! updated, please check again